flasc/energy_ratio/energy_ratio_output.py

```diff
@@ -54,6 +55,7 @@ def __init__(self,
             wd_cols (List[str]): The column names of the wind directions.
             ws_cols (List[str]): The column names of the wind speeds.
             uplift_cols (List[str]): The column names of the uplifts.
+            total_uplift_result (dict): Results of total uplift for each name in uplift_cols
```

Comment:
suggest describing what is in this dictionary, specifically what is contained in the results tuple